### PR TITLE
Conglomerate of Create methods

### PR DESCRIPTION
**Pull request by ajminer0913**

### CODE REVIEW

File: Create.java [hunk 1]
```diff
@@ -1,15 +1,183 @@
 import java.sql.Connection;
 import java.sql.ResultSet;
 import java.sql.Statement;
-public class Create {
+import java.sql.Connection;
```
Comment (Matthew-Bustamante) [inline Create.java:4]: Don't need this on line 4

File: Create.java [hunk 2]
```diff
@@ -1,15 +1,183 @@
...
+import java.sql.ResultSet;
```
Comment (Matthew-Bustamante) [inline Create.java:5]: Don't need this either on line 5

File: Create.java [hunk 3]
```diff
@@ -1,15 +1,183 @@
...
+
```
Comment (Matthew-Bustamante) [inline Create.java:6]: Need to import scanner.  I ran this on my machine and there was an error because there was no scanner

**Review by Matthew-Bustamante — CHANGES_REQUESTED** [review]
Overall looks good and everything works the way its suppose to.  I tested the validity of input statements and they do prevent words and letters from being entered but they do not prevent negative numbers from being entered.  This can be implemented later on but I just wanted to bring that to your attention.  Just make these simple changes and I'll approve.

**Comment by Matthew-Bustamante** [comment]
Disregard the previous comment, Nathan will make the minor changes.

**Review by Matthew-Bustamante — APPROVED** [review]
Disregard the previous comment, Nathan will make the minor changes. Forgot to approve

**Review by nkauffm1 — APPROVED** [review]
Besides the minor changes it looks good. I will make those changes and submit a pull request with those changes.

**Pull request merged**